### PR TITLE
Update Release Info

### LINKED ISSUE #6753: [STY Batch#2] Update Simorgh supported pages docs
**Is your feature request related to a problem? Please describe.**
Now that batch#2 of STY's are live we should update the docs in Simorgh that shows what pages are supported and what environment they are on.

Batch#2

```
kyrgyz,
igbo,
pashto,
sinhala,
Yoruba
```

**Describe the solution you'd like**
Update https://github.com/bbc/simorgh/blob/latest/docs/Simorgh-Release-Info.md to reflect the new live services for STY's from batch#2 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Resolves #6753

**Overall change:**
_Update Release Info_

**Code changes:**

- _Add release dates for batch 2 STYs to `scripts/simorghLaunchDates.js`._
- _Add batch 2 STY links for different enviroments to `cypress/support/config/settings.js`._
- _Run `node scripts/simorghPages.js` to update `docs/Simorgh-Release-Info.md`._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by thekp — APPROVED** [review]

File: cypress/support/config/settings.js [hunk 1]
```diff
@@ -2586,11 +2586,11 @@ module.exports = () => ({
...
-            paths: [],
+            paths: ['/kyrgyz/world-52917194'],
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/kyrgyz/world-23090100'],
             enabled: false,
           },
           local: {
```
Comment (amywalkerdev) [inline cypress/support/config/settings.js]: Can we add a local url too?

File: cypress/support/config/settings.js [hunk 2]
```diff
@@ -4514,11 +4514,11 @@ module.exports = () => ({
...
-            paths: [],
+            paths: ['/sinhala/world-52909487'],
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/sinhala/sri-lanka-23037210'],
             enabled: false,
           },
           local: {
```
Comment (amywalkerdev) [inline cypress/support/config/settings.js]: Same for Sinhala?

**Review by amywalkerdev — CHANGES_REQUESTED** [review]

File: cypress/support/config/settings.js (hunk 2 above)
Comment (FK78) [reply cypress/support/config/settings.js]: Added [here](https://github.com/bbc/simorgh/pull/6788/commits/4767d5abefd627065a1a1f550058ea7a25951316).

File: cypress/support/config/settings.js (hunk 1 above)
Comment (FK78) [reply cypress/support/config/settings.js]: Added [here](https://github.com/bbc/simorgh/pull/6788/commits/4767d5abefd627065a1a1f550058ea7a25951316).

Comment (karinathomasbbc) [reply cypress/support/config/settings.js]: I think we need to regenerate the release info after adding the local URLs 👍

**Review by amywalkerdev — APPROVED** [review]
Looks good! 👍

**Pull request merged**